docs/community/content/contribute/establish-project.en.md

```diff
@@ -0,0 +1,124 @@
++++
+title = "Establish project and environmental guidelines"
+weight = 2
+chapter = true
++++
+
+[TOC]
+
+# Establish the ShardingSphere software environment locally and obtain the latest installation package
+
+## Software environment
+
+
+
+| **Software** | **JDK** | **Maven** | **Idea**     |
+|  :--:  | :--:  |:--:  |  :--:  |
+| **Version** |  >=1.8  |  >=3.6.0  | The latest version |
+
+## Installation Procedure (For example, Mac)
+
+
+
+## 1.The JDK installation
+
+
+- The following link is to obtain the installation package suitable for your environment (MAC select.dmg format).
+	https://www.oracle.com/java
+
+![image20210818115720737](https://note.youdao.com/yws/public/resource/ae044dae27b19d4d2f191c76e6df0a10/xmlnote/WEBRESOURCE98df8ecd49565fed4be4dfe594d04521/17060)
+
+- Install it directly after downloading
+
+## 2.Maven environment deployment
```

Comment:
Using Maven Wrapper in our project is more convenient.

Reply:
@tristaZero  Please help to check whether maven related content needs to be removed from the content